apps/web/components/team/MemberChangeRoleModal.tsx

```diff
@@ -37,6 +38,23 @@ export default function MemberChangeRoleModal(props: {
       memberId: props.memberId,
       role,
     });
+
+    if (role === "OWNER") {
+      const members = props.team["members"];
+      let owner;
+
+      for (let i = 0; i < members.length; i++) {
+        if (members[i].role === "OWNER") {
+          owner = members[i].id;
+        }
+      }
+
```

Comment:
@zomars 
I know it probably won't happens but what should happens if no OWNER was found?
throw an owner not found exception would be fine